docs/decision_record.md

```diff
@@ -6,6 +6,42 @@
 
 [TOC]
 
+## 2021-11-17 - Structure for the return value of the comparison API endpoint
+
+### Decision
+
+At a minimum, the return value MUST return answers to basic questions, like "is one seqcol a subset of another", following the REQUIRED format specified below.
+
+**REQUIRED**: The endpoint MUST return, in JSON format, an object with these 7 keys: "any-elements-shared", "all-a-in-b", "all-b-in-a", "order-match", "no-overlap", "only-in-a", and "only-in-b". The value of each element should include a *list* of the attributes that meet the stated condition. 
+
+Example: 
+
+```
+{
+  "any-elements-shared": ["lengths", "names"],
+  "no-elements-shared": ["sequences"],  
+  "all-a-in-b": ["lengths", "names"],
+  "all-b-in-a": ["lengths", "names"],
+  "order-match": ["lengths", "names"],
```

Comment:
I think we need to specify exactly what we mean with `order-match`. Does this mean that the elements that are shared between a and b are in the same order, disregarding the rest, or does this imply that the arrays also have exactly the same content. I would assume it is the former, but it could easily be misunderstood.

Reply:
Yes, it is the former. You have to take this in context with the other elements, which specify to what degree there is overlap (any elements shared, no elements shared, all-x-in-y).

So, order match means exactly what you said: For elements that are shared, are they in the same order?

Is there a better way to say it? `shared-element-order-match` ?